torch/csrc/stable/ops.h

```diff
@@ -36,6 +36,15 @@ inline Tensor fill_(const Tensor& self, double value) {
   return self;
 }
 
+// We expect this to be the stable version of the amax.default op
+// with identical semantics to the existing amax.default op.
+inline Tensor amax(Tensor& self, int64_t dim[], int dim_len, bool keepdim) {
```

Comment:
@samanklesaria This change is unnecessary; `native_functions.yaml` specifies that `dim` is always of length 1, and there is no `dim_len` parameter in the `amax` function signature. What you need here is:

```suggestion
// dim is passed as a pointer to match the len-1 array semantics of the existing op.
inline Tensor amax(Tensor& self, int64_t* dim, bool keepdim) {
```

Reply:
@samanklesaria Never mind, it appears that the `native_functions.yaml` signature is incorrect; see https://github.com/pytorch/pytorch/pull/85911.